tests/installation/src/utils.ts

```diff
@@ -367,3 +367,14 @@ export async function showZoweRuntimeLogs(serverId: string, extraVars: {[key: st
     debug(`showZoweRuntimeLogs failed: ${e}`);
   }
 };
+
+export function identifySecuritySystem(): string {
```

Comment:
I think we don't need to identify security system from test cases based on `TEST_SERVER` name, the `zos_security_system` variable can be configured per host. By keeping `marist` to use the default value `RACF`, we can add `playbooks/host_vars/marist-2.yml` with content of `zos_security_system: ACF2` and `playbooks/host_vars/marist-3.yml` with content of `zos_security_system: TSS`. This way, it's easier to extend when we have more ACF2/TSS servers.

Just we need to remove the marist line from `playbooks/.gitignore`, which was added to avoid accidentally publish server password.